docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -1068,76 +1068,76 @@
     </sect2>
   </sect1>
 
-  <sect1 id="MoreHelmANDYAML" role="nobrk">
-    <title>More Helm and Yaml</title>
-
-    <para>This section is intended to provide some helpful information to get
-    started with a containerized deployment. There are numerous resources for
-    using Kubernetes, Helm, and Yaml files. Previously, we touched on the
-    <emphasis>values.yaml</emphasis> file and the values-schema.json file.
-    This section expands on some of those concepts and how they might be
-    applied when using the containerized version of the HPCC Systems platform.
-    For more information about using Kubernetes, Helm, or YAML files, or for
-    cloud or container deployments, refer to the respective
+  <sect1 id="MoreHelmANDYAML">
+    <title>Helm and Yaml Basics</title>
+
+    <para>This section is intended to provide some basic information to help
+    you in getting started with your HPCC Systems containerized deployment.
+    There are numerous resources available for learning about Kubernetes,
+    Helm, and YAML files. For more information about using these tools, or for
+    cloud or container deployments, please refer to the respective
     documentation.</para>
 
+    <para>In the previous section, we touched on the
+    <emphasis>values.yaml</emphasis> file and the
+    <emphasis>values-schema.json</emphasis> file. This section expands on some
+    of those concepts and how they might be applied when using the
+    containerized version of the HPCC Systems platform.</para>
+
     <sect2 id="TheValuesYamlFileStruct">
       <title>The <emphasis>values.yaml</emphasis> file structure</title>
 
-      <para>The <emphasis>values.yaml</emphasis> file is a yaml file. Yaml is
-      a data serialization language often used as a format for configuration
-      files. The construct that makes up the bulk of a yaml file is the
-      key-value pair, sometimes referred to as a hash or a dictionary. The
-      key-value pair construct consists of a key that points to some value(s).
-      The values could be strings, numbers, booleans, integers, arrays, or
-      dictionaries, and lists. These values are defined by the schema.</para>
+      <para>The <emphasis>values.yaml</emphasis> file is a YAML file which is
+      a format frequently used for configuration files. The construct that
+      makes up the bulk of a YAML file is the key-value pair, sometimes
+      referred to as a hash or a dictionary. The key-value pair construct
```

Comment:
trivial/not new: if you're going to mention "hash", it should really say "hash table", though I'd be inclined not to mention it at all and only say "sometimes referred to as a dictionary" ( "hash table" is more commonly used to describe the implementation )